helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java

```diff
@@ -546,8 +546,15 @@ int run(final Namespace options, final HeliosClient client, final PrintStream ou
       builder.setToken(token);
     }
 
-    builder.setAddCapabilities(options.<String>getList(addCapabilityArg.getDest()));
-    builder.setDropCapabilities(options.<String>getList(dropCapabilityArg.getDest()));
+    final List<String> addCaps = options.<String>getList(addCapabilityArg.getDest());
```

Comment:
For port mappings and registrations, it seems like we merge the values if they come from more than one source. Perhaps do the same here for consistency?